capstone/src/main/webapp/script.js

```diff
@@ -97,61 +106,42 @@ function createMap() {
           elementType: 'labels.text.stroke',
           stylers: [{color: '#17263c'}]
       }
-      ]
-    }
-  );
-
-  setMarkers(map);
+    ]
+  });
+  getSearchResults();
 }
 
-window.onload = createMap;
-
-// Data for the markers consisting of a name, a LatLng and a zIndex for the
-// order in which these markers should display on top of each other.
-var restaurants = [
-  {
-    "name": 'Tiny\'s & The Bar Upstairs', 
-    "latitude": 40.716778, 
-    "longitude": -74.00822, 
-    "zIndex": 1
-  },
-  {
-    "name": 'Max', 
-    "latitude": 40.71673, 
-    "longitude": -74.00828, 
-    "zIndex": 2
-  },
-  {
-    "name": 'Brooklyn Chop House', 
-    "latitude": 40.711334, 
-    "longitude":  -74.00632, 
-    "zIndex": 3
-  },
-];
+function getSearchResults() {
+  var request = {
+    location: center,
+    radius: 10000,
+    types: ["restaurant", "food"]
+  };
+  service = new google.maps.places.PlacesService(map);
+  service.nearbySearch(request, callback);
+}
 
-function setMarkers(map) {
-  // Adds markers to the map.
-  for (let restaurant of restaurants) {
-    var marker = new google.maps.Marker({
-        map: map,
-        draggable: true,
-        animation: google.maps.Animation.DROP,
-        position: {lat: restaurant.latitude, lng: restaurant.longitude},
-        zIndex: restaurant.zIndex
-    });
-    marker.addListener('click', toggleBounce);
-    marker.addListener('click', displayPanel(restaurant.name));
-    
-    marker.setMap(map);
+function callback(results, status) {
+  if (status == google.maps.places.PlacesServiceStatus.OK) {
+    for (var i = 0; i < results.length; i++) {
+      // TODO: change condition to check if matches filter
```

Comment:
Can we add a Github issue to the todo? For instance TODO(issuenum):  update condition to check if matches filter